Source/UnitTests/Core/PatchAllowlistTest.cpp

```diff
@@ -67,12 +70,22 @@ TEST(PatchAllowlist, VerifyHashes)
     Common::IniFile ini_file;
     ini_file.Load(file.physicalName, true);
     std::string game_id = file.virtualName.substr(0, file.virtualName.find_first_of('.'));
+    std::map<std::string /*hash*/, std::string /*name*/> hashes;
```

Comment:
It would be a bit more efficient to, instead of having an `std::map` where you store all hashes, have a "check hash" function that you can call from the patches loop, the AR codes loop and the Gecko loop. That way, instead of having to store the hash from each loop iteration in the `std::map`, you can discard the hash at the end of each loop iteration, like the code was doing before this PR.

Reply:
Ugh, I hate how messy this is looking, but as you wish.